simphony/cuds/abc_modeling_engine.py

```diff
@@ -16,6 +16,8 @@ class ABCModelingEngine(object):  # pragma: no cover
         container of attributes related to the computational method
     SP : DataContainer
         container of attributes related to the system parameters/conditions
+    materials : Materials
```

Comment:
There is some open question on where the materials should be stored (and how they should be accesed).   Right now,  I have the following
```
material_foo = Material(name="foo")
material.data[CUBA.MASS] = 1.0 
engine.materials.add_material(material_foo)
```
See #227 and https://github.com/simphony/simphony-common/issues/227#issuecomment-159869043

Any thoughts/ideas/critique would be appreciated.